src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
-import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Film {
+  [key: string]: string;
+}
+
+type Props = {
+  films: Film[];
+};
+
+export const MoviesList: React.FC<Props> = ({ films }: Props) => (
+  <>
```

Comment:
```suggestion
```
Why do you need this fragment here? This component already returns only one element - div